### ISSUE TITLE
[Screen reader - Logic Apps - Integration Accounts]: "Project details" & "Instance details" are not programmatically defined as heading.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6110]
### Describe the issue 

User Experience:

 Users who rely on screen reader for navigation will be impacted if headings are not defined programmatically as users will not get the heading information and they will not get the purpose of the content.

Note: User credentials should NOT be included in the bug.

Repro Steps:

- Open [https://ms.portal.azure.com](<https://ms.portal.azure.com>) and login via valid credentials. 
- Navigate to the Left Nav bar and select '+New', search 'Integration Accounts' and select it. Press the Create button. 
- Navigate all the controls and Enter details on the Integration Account blade and Press Create Button. 

## \- Verify whether "Project details" & "Instance details" are programmatically defined as heading. 

Actual Result:

 "Project details" & "Instance details" are not programmatically defined as heading.

Expected Result:

 "Project details" & "Instance details" should be programmatically defined as h3 heading.

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
https://msazure.visualstudio.com/One/_workitems/edit/29400599
